app/assets/stylesheets/application.css.scss

```diff
@@ -0,0 +1,366 @@
+
+#nav.affix {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index:10;
+}
+
+.navbar.scrolled {
+  background: rgb(255, 255, 255); //IE
+  background: rgba(248, 248, 248, 0.9); //NON-IE
+  -webkit-transition: all 0.6s ease;
+  -moz-transition: all 0.6s ease;
+  -o-transition: all 0.6s ease;
+  -ms-transition: all 0.6s ease;
+  transition: all 0.6s ease;
+}
+
+.nav-center {
+  margin:0;
+  float:none;
+}
+
+.navbar-inner{
+  text-align:center;
+}
+
+.navbar {
+  position: relative;
+  background: #FFFFFF;
+  -webkit-transition: all 0.6s ease;
+  -moz-transition: all 0.6s ease;
+  -o-transition: all 0.6s ease;
+  -ms-transition: all 0.6s ease;
+  transition: all 0.6s ease;
+}
+
+.navbar-right {
+  margin-right: 0px;
+}
+
+.brand {
+  position: absolute;
+  left: 50%;
+  margin-left: -275px !important;  /* 50% of your logo width */
+  margin-top: -52px;
+  display: block;
+}
+
+.jumbotron {
+   height: 200px;
+   background: #FFFFFF;
+   margin-bottom: 0px;
+}
+
+.logo {
+   position: absolute;
+   left: 0px;
+   top: 24px;
+}
+
+.main-content {
+  background: #000000;
+  background-image: image-url("galaxy.jpeg");
+  background-size: cover;
+  background-attachment: fixed;
+}
+
+.video {
+  text-align: center;
+  margin-bottom: 10px;
+  iframe {
+    border-left: 0px;
+    border-right: 0px;
+    margin-left: -25px;
+  }
+}
+
+.row-centered {
+  text-align: center;
+  margin-left: 25px;
+}
+
+.fm-row-centered {
+  text-align: center;
+}
+
+@media (min-width: 1200px){
+  .fm-row-centered {
+    margin-left: 90px;
+  }
+}
+
+.c-search {
+  background-color: rgba(255, 255, 255, 0.9);
+  border-radius: 15px;
+  margin: 15px;
+  padding: 10px;
+  text-align: center;
+  display:inline-block;
+  width: 240px;
+  overflow: scroll;
+}
+
+@media (min-width: 992px){
+  .c-search {
+    height: 145px;
+  }
+}
+
+.display-img {
+  border-radius: 50%;
+}
+
+.form-control:focus {
+  border-color: #EDA83C;
+  box-shadow: 0px 1px 1px #EDA83C inset, 0px 0px 8px #EDA83C;
+}
+
+:focus  {
+  outline: 0 !important;
+  outline-color: #EDA83C;
+  border-color: #EDA83C;
+  box-shadow: 0px 1px 1px #EDA83C inset, 0px 0px 8px #EDA83C;
+}
+
+a:focus {
+  outline-color: none;
+  border-color: #EDA83C;
+  box-shadow: 0px 1px 1px #EDA83C inset, 0px 0px 8px #EDA83C;
+}
+
+.login {
+  margin-top: 50px;
+  text-align: center;
+}
+
+.twitter-color {
+  color: #55acee;
+}
+
+.vimeo-color {
+  color: #1ab7ea;
+}
+
+.feed-manager {
+  background-color: rgba(255, 255, 255, 0.9);
+  border-radius: 15px;
+  margin: 15px;
+  padding: 10px;
+  text-align: center;
+  display:inline-block;
+  width: 270px;
+}
+
+h3 {
+  margin-top: 120px;
+  color: white;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  font-size: 30px;
+}
+
+// CSS Loader
+#floatingCirclesG{
+	position:relative;
+	width:125px;
+	height:125px;
+	margin:auto;
+	transform:scale(0.6);
+		-o-transform:scale(0.6);
+		-ms-transform:scale(0.6);
+		-webkit-transform:scale(0.6);
+		-moz-transform:scale(0.6);
```

Comment:
I think these should be at the same indentation level as `transform:scale(0.6);`